### PR TITLE
zh-translation: /docs/tasks/traffic-management/request-routing/index.md

### PR DESCRIPTION
**Pull request by ilylia**
ref: https://github.com/servicemesher/istio-official-translation/issues/1083

### CODE REVIEW

File: content/zh/docs/tasks/traffic-management/request-routing/index.md [hunk 1]
```diff
@@ -185,41 +168,32 @@ Remember, `reviews:v2` is the version that includes the star ratings feature.
...
-    to `reviews:v1` for all users except Jason.
+    刷新浏览器。现在星星消失了。这是因为除了 Jason 之外，所有用户的流量都被路由到 `reviews:v1`。
 
-You have successfully configured Istio to route traffic based on user identity.
+您已成功配置 Istio 以根据用户身份路由流量。
 
-## Understanding what happened
+## 理解原理
 
-In this task, you used Istio to send 100% of the traffic to the `v1` version
```
Comment (rootsongjc) [inline content/zh/docs/tasks/traffic-management/request-routing/index.md]: 这些英文原文都没删除？

**Comment by rootsongjc** [comment]
@ilylia Please sign the CLA first.

**Comment by ilylia** [comment]
@googlebot I signed it!

File: content/zh/docs/tasks/traffic-management/request-routing/index.md (hunk 1 above)
Comment (ilylia) [reply content/zh/docs/tasks/traffic-management/request-routing/index.md]: 是的

> 这些英文原文都没删除？

是的

**Comment by rootsongjc** [comment]
>> 430 files are free from spelling errors
>> 429 files are free from spelling errors
./zh/docs/tasks/traffic-management/request-routing/index.md:46: MD029 Ordered list item prefix
./zh/docs/tasks/traffic-management/request-routing/index.md:111: MD029 Ordered list item prefix
./zh/docs/tasks/traffic-management/request-routing/index.md:144: MD029 Ordered list item prefix
./zh/docs/tasks/traffic-management/request-routing/index.md:171: MD029 Ordered list item prefix
./zh/docs/tasks/traffic-management/request-routing/index.md:175: MD029 Ordered list item prefix
./zh/docs/tasks/traffic-management/request-routing/index.md:199: MD029 Ordered list item prefix
A detailed description of the rules is available at https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md
Running ["LinkCheck", "HtmlCheck", "OpenGraphCheck", "ImageCheck", "ScriptCheck"] on ["./public"] on *.html... 
Found 0 links in the cache...
Adding 890 links to the cache...
Removing 0 links from the cache...
Checking 890 external links...
Ran on 760 files!
- ./public/zh/docs/tasks/traffic-management/request-routing/index.html
  *  internally linking to /zh/docs/setup/kubernetes/additional-setup/requirements, which does not exist (line 53045)
     <a href="/zh/docs/setup/kubernetes/additional-setup/requirements">Pods 和 Services 需求</a>
  *  linking to /zh/docs/examples/bookinfo/#清理, but 清理 does not exist (line 53057)
     <a href="/zh/docs/examples/bookinfo/#%E6%B8%85%E7%90%86">Bookinfo 清理</a>
  *  linking to /zh/docs/examples/bookinfo/#确定-ingress-的-ip-和端口, but 确定-ingress-的-ip-和端口 does not exist (line 52980)
     <a href="/zh/docs/examples/bookinfo/#%E7%A1%AE%E5%AE%9A-ingress-%E7%9A%84-ip-%E5%92%8C%E7%AB%AF%E5%8F%A3">Bookinfo</a>
htmlproofer 3.12.0 | Error:  HTML-Proofer found 3 failures!
LINTING FAILED
Makefile.core.mk:27: recipe for target 'lint' failed
make: *** [lint] Error 1

- 请不要翻译锚点，所有标题后使用 `# 这是一个锚点 {#anchor}`格式，使用英文原有锚点，示例：
英文原文为：`## Before you begin`，则中文文档中应为 `## 开始之前{#before-you-begin}`。

File: content/zh/docs/tasks/traffic-management/request-routing/index.md [hunk 2]
```diff
@@ -1,55 +1,49 @@
 ---
-title: Request Routing
-description: This task shows you how to configure dynamic request routing to multiple versions of a microservice.
+title: 配置请求路由
+description: 此任务将展示如何将请求动态路由到微服务的多个版本。
 weight: 10
 aliases:
-    - /docs/tasks/request-routing.html
```
Comment (rootsongjc) [inline content/zh/docs/tasks/traffic-management/request-routing/index.md]: 英文的 aliases 路径可以删掉

File: content/zh/docs/tasks/traffic-management/request-routing/index.md [hunk 3]
```diff
@@ -1,55 +1,49 @@
...
+    - /zh/docs/tasks/request-routing.html
 keywords: [traffic-management,routing]
 ---
 
-This task shows you how to route requests dynamically to multiple versions of a
-microservice.
+此任务将展示如何将请求动态路由到微服务的多个版本。
 
-## Before you begin
+## 开始之前
```
Comment (rootsongjc) [inline content/zh/docs/tasks/traffic-management/request-routing/index.md]: 所有的中文标题都要补充英文锚点，如：`开始之前{#before-you-begin}`

File: content/zh/docs/tasks/traffic-management/request-routing/index.md [hunk 4]
```diff
@@ -1,55 +1,47 @@
 ---
-title: Request Routing
-description: This task shows you how to configure dynamic request routing to multiple versions of a microservice.
+title: 配置请求路由
+description: 此任务将展示如何将请求动态路由到微服务的多个版本。
 weight: 10
-aliases:
-    - /docs/tasks/request-routing.html
```
Comment (rootsongjc) [inline content/zh/docs/tasks/traffic-management/request-routing/index.md]: `/zh/docs/tasks/request-routing.html` 还要保留的。

Comment (ilylia) [reply content/zh/docs/tasks/traffic-management/request-routing/index.md]: 已修改，等lint #998

**Pull request merged**